k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -744,7 +713,7 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             mMessageContentView.requestFocus();
         } else {
             // Explicitly set focus to "To:" input field (see issue 2998)
-            mToView.requestFocus();
+            recipientView.toFieldRequestFocus();
```

Comment:
Put the action first when naming methods, e.g. `requestFocusOnToField()`.
On first glance `to…` as method name gives the impression the object is converted into some other format.